tmk_core/protocol/ps2_mouse.h

```diff
@@ -75,14 +75,24 @@ __attribute__((unused)) static enum ps2_mouse_mode_e {
  *    1|[                    X movement(0-255)                         ]
  *    2|[                    Y movement(0-255)                         ]
  */
-#define PS2_MOUSE_BTN_MASK 0x07
-#define PS2_MOUSE_BTN_LEFT 0
-#define PS2_MOUSE_BTN_RIGHT 1
-#define PS2_MOUSE_BTN_MIDDLE 2
-#define PS2_MOUSE_X_SIGN 4
-#define PS2_MOUSE_Y_SIGN 5
-#define PS2_MOUSE_X_OVFLW 6
-#define PS2_MOUSE_Y_OVFLW 7
+#define PS2_MOUSE_BTN_MASK 0x07 // 0b0000111
+#define PS2_MOUSE_BTN_LEFT 0    // 0b0000001
+#define PS2_MOUSE_BTN_RIGHT 1   // 0b0000010
+#define PS2_MOUSE_BTN_MIDDLE 2  // 0b0000100
+#define PS2_MOUSE_X_SIGN 4      // 0b0001000
+#define PS2_MOUSE_Y_SIGN 5      // 0b0010000
+#define PS2_MOUSE_X_OVFLW 6     // 0b0100000
+#define PS2_MOUSE_Y_OVFLW 7     // 0b1000000
```

Comment:
clang-format
```suggestion
#define PS2_MOUSE_BTN_MASK 0x07  // 0b0000111
#define PS2_MOUSE_BTN_LEFT 0     // 0b0000001
#define PS2_MOUSE_BTN_RIGHT 1    // 0b0000010
#define PS2_MOUSE_BTN_MIDDLE 2   // 0b0000100
#define PS2_MOUSE_X_SIGN 4       // 0b0001000
#define PS2_MOUSE_Y_SIGN 5       // 0b0010000
#define PS2_MOUSE_X_OVFLW 6      // 0b0100000
#define PS2_MOUSE_Y_OVFLW 7      // 0b1000000
```

Reply:
Yes. I know. I used just to remember it. I will put WIP when I make a test to clarify this.